.neuro/live.yml

```
@@ -0,0 +1,50 @@
+kind: live
+title: Fooocus
+
+defaults:
+  life_span: 5d
+  preset: a100x1
+
+images:
+  fooocus:
+    ref: ghcr.io/neuro-inc/fooocus:latest
+  fooocus_apolo:
+    ref: image:${{ flow.project_id }}
+    context: ${{ flow.workspace }}
+    dockerfile: ${{ flow.workspace }}/Dockerfile
+    build_preset: cpu-medium
+volumes:
+  models:
+    remote: storage:/$[[ project.project_name ]]/$[[ flow.project_id ]]/data/models
+    mount: /content/data/model
+    local: models
+  outputs:
+    remote: storage:/$[[ project.project_name ]]/$[[ flow.project_id ]]/app/outputs
+    mount: /content/app/outputs
+
+jobs:
+  fooocus:
+    # image: ${{images.fooocus_apolo.ref}}
+    image: ${{images.fooocus.ref}}
+    name: fooocus
+    preset: a100x1
+    http_port: 7865
+    browse: true
+    volumes:
+      - ${{volumes.models.ref_rw}}
+      - ${{volumes.outputs.ref_rw}}
+    env:
+      CMDARGS: --listen # Arguments for launch.py.
+      DATADIR: /content/data # Directory which stores models, outputs dir
+      config_path: /content/data/config.txt
+      config_example_path: /content/data/config_modification_tutorial.txt
+      path_checkpoints: /content/data/models/checkpoints/
```

here you could use

```suggestion
      path_checkpoints: ${{ volumes.data.mount }}/checkpoints/
```

see https://docs.apolo.us/apolo-flow-reference/workflow-syntax/live-workflow-syntax/live-contexts#volumes-context

fyi @neuro-inc/mlops

---

done!